topics/persistence.md

```diff
@@ -68,6 +68,17 @@ dataset to disk every 60 seconds if at least 1000 keys changed:
 
 This strategy is known as _snapshotting_.
 
+## No persistence
+
+If you prefer **not** to have persistence (eg. when using a Valkey instance solely as a cache) that is also a possibility.
```

Comment:
The spellchecker complains about "eg." I think it wants it spelled "e.g.", but I recall now that we've avoided abbreviatinos like this in the docs in the past. Maybe "for example" is clearer?

```suggestion
If you prefer **not** to have persistence (for example when using a Valkey instance solely as a cache) that is also a possibility.
```

Reply:
Agreed my bad. I also also reluctant.